### PR TITLE
update(libsinsp): double thread_table_absolute_max_size

### PR DESCRIPTION
**Pull request by incertum**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

/kind design

> /kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area API-version

> /area build

> /area CI

> /area driver-kmod

> /area driver-bpf

> /area driver-modern-bpf

> /area libscap-engine-bpf

> /area libscap-engine-gvisor

> /area libscap-engine-kmod

> /area libscap-engine-modern-bpf

> /area libscap-engine-nodriver

> /area libscap-engine-noop

> /area libscap-engine-source-plugin

> /area libscap-engine-savefile

> /area libscap

> /area libpman

/area libsinsp

> /area tests

> /area proposals

**Does this PR require a change in the driver versions?**

> /version driver-API-version-major

> /version driver-API-version-minor

> /version driver-API-version-patch

> /version driver-SCHEMA-version-major

> /version driver-SCHEMA-version-minor

> /version driver-SCHEMA-version-patch

**What this PR does / why we need it**:

Deployed the new libsinsp state metrics and for the first time we have visibility into drops due to a full threadtable (note this metric was never exposed before not even via the old libsinsp stats). It turns out that the legacy default absolute max value does not seem to cut it anymore on more modern infrastructure. 

Went ahead and deployed this patch to production and like a miracle no more drops at all even on busy clusters. Looked at CPU and memory metrics and it seems safe. I observed metrics for over a week. By the way the state engine is healthy in many regards, I just observed some puzzling correlation between store events actions and higher memory usage. However, higher memory usage seems not related to the plain number of threads we store. Let me open a dedicated issue to discuss this further no not digress too much here.

Of course we could expose the absolute max threadtable size as config, not sure however if it is needed given we already have so many configs.

In summary, again global memory on a very large fleet did not change after increasing this value, this is why I would like to say that it seems safe with some level of confidence. Needless to say how much we gain from no drops due to a full threadtable.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by jasondellaluce — APPROVED** [review]
/approve

Works for me. I like the idea of making this configurable tho.

**Review by FedeDP — APPROVED** [review]
Agree with Jason about making this configurable. Anyway, LGTM!
/approve
